src/components/ToolsMenuItem.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {Tooltip} from 'react-tippy'
+import Icon from './ui/Icon'
+import '../styles/ToolsMenuItem.scss'
+import {EIconType} from '../enums/EIconName'
+
+interface IProps {
+    action: () => void,
+    disabled?: boolean
+    tooltip?: string,
+    iconName: string,
+    className?: string
+}
+
+const ToolsMenuItem = (props: IProps) => {
+    const {iconName, action, tooltip, disabled, className} = props
+
+    return (
+        <div className={'scope__ToolsMenuItem'}>
+            <Tooltip
+                title={tooltip}
+                position={'bottom'}
+                arrow={true}
+                duration={200}
+                theme={'transparent'}
+                size={'small'}
+                arrowSize={'small'}
+            >
+                <button
+                    className={`tools-menu__item ${className ? `tools-menu__item--${className}` : ''}`}
+                    onClick={action}
+                    disabled={disabled}
+                >
+                    <span className={'main-icon'}>
+                        <Icon name={iconName} type={EIconType.SOLID}/>
```

Comment:
v UIKitu jsem toto pojmenoval `prefix`, ale to je asi jedno, jen tip na zlepseni pripadne

zvazil bych dat nejakou default value pro type, tady se mi zda se ti dost casto opakuje SOLID, navic kdyz jine definovane nemas